src/core/lwip/tcp_out.c

```diff
@@ -1537,6 +1552,11 @@ void tcp_split_segment(struct tcp_pcb *pcb, struct tcp_seg *seg, u32_t wnd)
             return;
         }
 
+        if (seg->p->desc.attr == PBUF_DESC_EXPRESS) {
```

Comment:
What about tcp_split_one_segment and tcp_split_rexmit?

Reply:
They're removed in #98. I didn't touch the code not to make conflicts. Probably need to merge the removal to vNext outside the PR.